src/components/AnimatedCircularProgress.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from "react";
+import { Animated, Easing, EasingFunction } from "react-native";
+import CircularProgress, {
+  Props as CircularProgressProps,
+} from "./CircularProgress";
+const AnimatedProgress = Animated.createAnimatedComponent(CircularProgress);
+
+interface Props extends CircularProgressProps {
+  duration?: number;
+  easing?: EasingFunction;
+  prefill?: number;
+  useNativeDriver?: boolean;
+  tintColorSecondary?: string;
+  onAnimationComplete?: Animated.EndCallback | undefined;
+}
+
+const AnimatedCircularProgress: React.FC<Props> = ({
+  duration = 500,
+  easing = Easing.out(Easing.ease),
+  fill,
+  prefill = 0,
+  useNativeDriver = false,
+  tintColorSecondary,
+  onAnimationComplete,
+  tintColor = "black",
+  ...other
+}) => {
+  const [fillAnimation] = React.useState<Animated.Value>(
+    new Animated.Value(prefill)
+  );
+
+  //   const reAnimate = (prefill, toVal, dur, ease) => {
```

Comment:
im assuming this doesn't work yet?

Reply:
was in original component, never called anywhere